src/App.css

```diff
@@ -0,0 +1,254 @@
+  margin-top: -14px;
+}
+
+.main-conteiner ul {
+  display: flex;
+  gap: 32px;
+  list-style-type: none;
+  align-items: center;
+  padding: 0 2%;
+}
+
+.header-icon {
+  display: flex;
+  justify-content: flex-end;
+  width: 64%;
+}
+
+.header-icon img {
+  border: 1px solid var(--border-color);
+  border-radius: 34px;
+  cursor: pointer;
+  padding: 8px;
+  width: 44px;
+}
+
+.header-title {
+  font-family: "Montserrat", sans-serif;
+  font-size: 1.875rem;
+  font-weight: 700;
+  color: var(--primary-color);
+  padding-left: 5%;
+  white-space: nowrap;
+}
+
+.header-main a {
+  opacity: 1;
+  font-family: "Montserrat", sans-serif;
+  font-size: 0.813rem;
+  color: var(--text-color);
+  letter-spacing: 1.9px;
+  text-decoration: none;
+}
+
+hr {
+  height: 0.125rem;
+  border: 1px solid var(--border-color);
+  width: 100%;
+}
+
+form {
+  display: flex;
+  padding: 0 2%;
+  justify-content: flex-start;
+  gap: 17px;
+}
+
+.form-main {
+  padding: 30px;
+}
+
+.form-main h2 {
+  font-size: 1.25rem;
+  font-weight: 700;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+#title,
+#author {
+  padding: 12px;
+  border-radius: 4px;
+  border: 1px solid var(--border-color);
+  background-color: #fff;
+  font-size: 1rem;
+  letter-spacing: -0.15px;
+  color: var(--text-color);
+  width: 22rem;
+}
+
+#title::placeholder,
+#author::placeholder {
+  opacity: 0.5;
+  color: #121212;
+}
+
+.form-main button {
+  cursor: pointer;
+  width: 20%;
+  padding: 12px;
+  border-radius: 3px;
+  background-color: var(--primary-color);
+  border: none;
+  letter-spacing: 0.5px;
+  color: #fff;
+  font-size: 1rem;
+  font-weight: 700;
+  font-family: "Roboto Slab", serif;
+}
+
+.book-progress {
+  width: 5.25rem;
+  height: 5.25rem;
+  border-radius: 50%;
+  border: 5px solid #307bbe;
+  border-top: 5px solid var(--border-color);
+  transform: rotate(-45deg);
+  margin-top: 15px;
+}
+
+.book-collection {
+  display: flex;
+  padding: 4%;
+  gap: 26px;
+  flex-direction: column;
+}
+
+.book-main {
+  display: flex;
+  padding: 32px;
+  border: 1px solid lightgray;
+  background: white;
+  height: 180px;
+  border-radius: 7px;
+}
+
+.book-details {
+  width: 42%;
+  word-wrap: break-word;
+}
+
+.book-percentage {
+  padding: 28px 2%;
+}
+
+.book-percentage p:first-child {
+  font-size: 37px;
+  margin: 0;
+  padding: 0;
+  font-family: "Montserrat", sans-serif;
+  color: var(--text-color);
+}
+
+.book-percentage p:last-child {
+  font-size: 0.875rem;
+  opacity: 0.5;
+  margin: 0;
+}
+
+.book-details p:first-child {
+  font-weight: 700;
+  color: var(--text-color);
+  font-family: "Montserrat", sans-serif;
+  font-size: 0.875rem;
+  opacity: 0.5;
+  margin: 1px;
+}
+
+.book-details h3 {
+  font-size: 1.375rem;
+  letter-spacing: -0.2px;
+  font-weight: 700;
+  color: var(--text-color);
+  margin: 1px;
+  font-family: "Roboto Slab", serif;
+}
+
+.book-details-text {
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #0290ff;
+  font-family: "Roboto Slab", serif;
+}
```

Comment:
Great job @Jrmcuervo 🤝

- [ ] As stated in the project requirements, kindly do not repeat the same rule, instead create a single class for the repeating rule and reuse this class in your jsx code. Kindly impelement this for `font-family: "Roboto Slab", serif;`, `font-family: "Montserrat", sans-serif;`, and `flex`.